src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -423,14 +423,6 @@
 				ToolTipService.ToolTip="{helpers:ResourceString Name=NavToolbarSelectionOptions/ToolTipService/ToolTip}">
 				<AppBarButton.Flyout>
 					<MenuFlyout Placement="Bottom">
-						<ToggleMenuFlyoutItem
-							x:Name="MultiselectMFI"
-							IsChecked="{x:Bind AppModel.MultiselectEnabled, Mode=TwoWay}"
-							Text="{helpers:ResourceString Name=NavToolbarMultiselect/Text}">
```

Comment:
Can you remove the string from the en-us file? Crowdin will sync the change to the other files.